test.js

```diff
@@ -674,7 +674,7 @@ test('uses the custom successMessage callback if passed in as an option', functi
   var dest = split(JSON.parse)
   var customResponseMessage = 'Custom response message'
   var logger = pinoHttp({
-    customSuccessMessage: function (res) {
+    customSuccessMessage: function (res, err) {
```

Comment:
Where is the test for the `err` possible values?

Reply:
@jsumners Added a test for the optional param